examples/platforms/utils/logging_rtt.h

```diff
@@ -54,8 +54,12 @@ extern "C" {
  *
  */
 #ifndef LOG_RTT_BUFFER_INDEX
+#if OPENTHREAD_UART_RTT_ENABLE
+#define LOG_RTT_BUFFER_INDEX 1
+#else
 #define LOG_RTT_BUFFER_INDEX 0
```

Comment:
Could we keep the RTT logging to use the channel 0, and set the RTT Uart to use channel 1 by default. So that the usage of the RTT client keeps the same when outputting logs.